lib/message.rb

```diff
@@ -0,0 +1,36 @@
+require 'pg'
+
+class Message
+
+  attr_reader :id, :message, :time
+
+  def initialize(id:, message:, time:)
+    @id = id
+    @message = message
+    @time = time
+  end
+
```

Comment:
I think I would prefer to put the raw time into the db.  It's a bit more flexible and if you want to search or filter on this field later you don't need to do any parsing.

Converting the time to a user friendly format is presentational and could be done in the view, or by a specific presenter class.

Doing this also means that if you change your mind about the design of the user interface (and the way that times are displayed there) you don't need to change everything in the posts table :)